js/interface.js

```diff
@@ -1241,8 +1241,10 @@ var DynamicLists = (function() {
       }
       if (context === 'relations') {
         $('.relations-tab').removeClass('present').addClass('future');
-
-        initDataSourceProvider(_this.config.dataSourceId);
+        
+        if (!dataSourceProvider) {
+          initDataSourceProvider(_this.config.dataSourceId);
```

Comment:
It would be more suitable for `initDataSourceProvider()` to make sure that if it's already initialised, don't do anything.

https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/6589c527fc010aba5f32ab9039564bcb59bdac4d/js/interface.js#L106

```js
function initDataSourceProvider(currentDataSourceId) {
  if (dataSourceProvider) {
    return;
  }

  // ...
}
```

Reply:
@tonytlwu , You are right, already fixed, thanks